### PR TITLE
Moved client version out of source.

### PR DESCRIPTION
**Pull request by TeoTwawki**

### CODE REVIEW

File: src/login/login.cpp [hunk 1]
```diff
@@ -368,6 +409,13 @@ int32 login_config_default()
 	return 0;
 }
 
+int32 version_info_default()
+{
+    version_info.Min_Client_Ver = "01010101_0"; // xxYYMMDD_m = xx:MajorRelease YY:year MM:month DD:day _m:MinorRelease
```
Comment (teschnei) [inline src/login/login.cpp]: put the default 99999999_9

Comment (TeoTwawki) [inline src/login/login.cpp]: k

**Comment by TeoTwawki** [comment]
@teschnei is all 9's now.

File: src/login/login.cpp [hunk 2]
```diff
@@ -38,11 +38,12 @@
...
 #include "message_server.h"
 
-const char *LOGIN_CONF_FILENAME = NULL;
+const char* LOGIN_CONF_FILENAME = NULL;
+const char* VERSION_INFO_FILENAME = NULL;
 
 //lan_config_t   lan_config;		// lan settings
 login_config_t login_config;	//main settings
-
+version_info_t version_info;
```
Comment (maxtherabbit) [inline src/login/login.cpp]: why a struct for this? it's just one string

Comment (TeoTwawki) [inline src/login/login.cpp]: Future use, and I didn't want it in the lobby conf either.

**Pull request merged**